extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -16,12 +16,14 @@ import * as trackIcons from '../icons/track-icons';
 import { STATE_ERROR, STATE_PLAYING } from '../constants';
 
 const TrackIcon = ( { isPlaying, isError, className } ) => {
-	let name;
+	let hiddenText, name;
 
 	if ( isError ) {
 		name = 'error';
+		hiddenText = __( 'Error:', 'jetpack' );
 	} else if ( isPlaying ) {
 		name = 'playing';
+		hiddenText = __( 'Playing: ', 'jetpack' );
```

Comment:
This has a trailing whitespace, the above doesn't. Is the colon needed?
It might be worth using `_x()` here to give translators more context about how this string is being used.
https://github.com/WordPress/gutenberg/blob/master/packages/i18n/README.md#_x

Reply:
Both now have a trailing whitespace and translators context.

Reply:
Sorry, I overlooked this when I last checked. Please don't use trailing whitespace in translatable strings, it will be stripped. Either add it just through HTML or give the `<span>` a padding.

Reply:
Can do! Very good to know. Thanks!

Reply:
Updated.